src/configureStore.js

```diff
@@ -78,11 +79,13 @@ const enhancer = composeWithDevTools({
   // Options: https://github.com/jhen0409/react-native-debugger#options
 })(applyMiddleware(...middlewares));
 
+const sentryReduxEnhancer = Sentry.createReduxEnhancer();
+
 const configureStore = (initialState: ?Object): Object => {
   const store = createStore(
     pReducer,
     initialState,
-    enhancer,
+    compose(enhancer, sentryReduxEnhancer),
```

Comment:
I think we should exclude some actions with the private data (private keys, mnemonic, etc) like it's described here: https://docs.sentry.io/platforms/javascript/guides/react/configuration/integrations/redux/#redux-enhancer-options

Reply:
P. S. Overall that change is awesome!

Reply:
I define one action but i am not sure about all actions that have sensitive data